src/components/Products/ProductArticle.js

```diff
@@ -0,0 +1,102 @@
+import { Card, Icon, Image, Text } from "../Reusable"
+
+const ProductArticle = ({ prd }) => {
+    const {
+
+        name,
+        company,
+        size,
+        price,
+        offerPrice,
+        img: {
+            srcSet,
+            alt,
+            sizes
+        },
+        rating,
+        quantity
+    } = prd
+
+    return (
+        <Card classes="pd-xs pos-relative">
+            {
+                offerPrice &&
+                <Text classes="pos-absolute tr-1 bg-warn txt-md txt-600 txt-primary txt-ucase pd-xs">offer</Text>
+            }
+            {
+                quantity < 15 && <Text classes='txt-cap pos-absolute tl-0 bg-err txt-md'>{`only ${quantity} left.`}</Text>
+            }
+
+            <Image srcSet={srcSet} alt={alt} sizes={sizes} />
+
+            <Text classes="txt-md txt-cap txt-primary">{company}</Text>
+
+            <Text classes="txt-md txt-cap txt-primary mg-btm-xs">{name}</Text>
+
+            <div className="flx mg-btm-xs">
+                <Text classes='txt-md txt-primary txt-cap mg-right-s'>sizes</Text>
+                {
+                    size.map(size => <Text key={size} classes='txt-md txt-primary mg-right-xs'>{size}</Text>)
+                }
+            </div>
+
+            <div className="flx">
+                {
+                    offerPrice ?
+                        <div className="flx flx-column">
+
+                            <div className="flx">
+
+                                <Text
+                                    classes="txt-cut txt-md txt-cap txt-off-secondary">rs.
```

Comment:
I would suggest instead of naming `classes` to props, name it `className` as you have done at other places. 
Currently, it seems like `classes` and `className` are two different things, but I guess, they are the same.

In the `Text` component, you can destructure `className` prop and then can use template literal to assign them to the `className` attribute.

Reply:
yes , i will update this part after merging all pages.